test/cli_run_working_dir_test.go

```diff
@@ -81,7 +81,11 @@ func (suite *PouchRunWorkingDirSuite) TestRunWithWorkingDir(c *check.C) {
 	dir := "/tmp/testworkingdir"
 
 	cname := "TestRunWithNotExistWorkingDir"
-	res := command.PouchRun("run", "-m", "100m", "-w", dir, "--name", cname, busyboxImage, "pwd")
+	// Some Kernel not support Memory, so result information may contains some
+	// warning information like:
+	// WARNING: Current Kernel does not support memory swap, discard --memory-swap
```

Comment:
I don't think we should add the `-m 100m` in this case, because we didn't check the memory. Please remove the comment.